### DISCUSSION TITLE
Possible to add an "Are you really, truly sure?" prompt when removing blog.

### DISCUSSION THREAD

**Discussion by Someonemustnothavethis** [TumblThreeApp/TumblThree#316] [Q&A (Question & Answer)]
Today, during my monthly update I hit 'Remove Blog' to get rid of one that had died.  Unfortunately it removed the highlighted ones I'd selected to crawl vice the one I wanted to remove.   I know it was my own fault, but having an "Are ya Sure?" prompt would have made me double check.
Thankfully it was near the top of the list so I only lost a half dozen.  I'll just add them again once the update is finished.
Thanks for considering this small request.
Rob and his filthy Tumblr habit.

**Comment by thomas694** [comment]
That's already possible.
You should enable the option "display confirmation dialogs" in the settings, maybe also "archive the index files instead of deleting".

The confirm dialog setting is now enabled by default for new installations.

**Comment by thomas694** [comment]
The issue has been fixed and closed. You can still comment. Feel free to ask for reopening the issue if needed.
